src/cdk/stack.ts

```diff
@@ -85,18 +85,24 @@ export interface ExportOutputs {
   addOutput (name: string, value: string, shouldExport: boolean): void
 }
 
+interface MiraStackProps {
+  disablePolicies?: boolean
+  approvedWildcardActions?: string[]
+  [x: string]: unknown
```

Comment:
Almost thinking these props are getting defined by `cdk.Stack`?  Am wondering if conflicts with the underlying type somehow?  Wondering if there is potential for conflict if inheriting from the underlying type is possible?

Disregard if not relevant.